addons/mail/static/src/components/thread_view/thread_view_tests.js

```diff
@@ -431,6 +431,91 @@ QUnit.test('show message subject if thread is mailing channel', async function (
     );
 });
 
+QUnit.test('message list for tracking field', async function (assert) {
+    assert.expect(3);
+    await this.start();
+    this.data["mail.message"].fields.tracking_value_ids = {
+        relation: "mail.tracking.values",
+        string: "Tracking Values",
+        type: "many2many",
+    }
+    const thread = this.env.models['mail.thread'].create({
+        channel_type: 'channel',
+        id: 100,
+        members: [['insert', [
+            {
+                email: "john@example.com",
+                id: 9,
+                name: "John",
+            },
+        ]]],
+        model: 'mail.channel',
+        name: "General",
+        public: 'public',
+    });
+
+    this.data['mail.message'].records.push({
+        channel_ids: [100],
+        model: 'mail.channel',
+        res_id: 100,
+        id: 1,
+        tracking_value_ids: [{
+            changed_field: "Field 1",
```

Comment:
Rename to just "Field" (if you split in 3 tests you just need 1 field per test).